libfs/fs.go

```diff
@@ -484,33 +486,81 @@ func (fs *FS) Readlink(link string) (target string, err error) {
 }
 
 // Chmod implements the billy.Filesystem interface for FS.
-func (fs *FS) Chmod(name string, mode os.FileMode) error {
-	return nil
+func (fs *FS) Chmod(name string, mode os.FileMode) (err error) {
+	fs.log.CDebugf(fs.ctx, "Chmod %s %s", name, mode)
+	defer func() {
+		fs.deferLog.CDebugf(fs.ctx, "Chmod done: %+v", err)
+	}()
+
+	n, _, err := fs.lookupOrCreateEntry(name, os.O_RDONLY, 0)
```

Comment:
Yep, exactly!  With just a bare `0`, there's no indication that `O_CREATE` is even an option.  With an explicit `O_RDONLY` (which is defined as 0, by the way), it seems more clear to me that creation is being explicitly ruled out.  Just a personal preference I guess.